src/plugins/usage_collection/common/constants.ts

```diff
@@ -7,5 +7,5 @@
  */
 
 export const KIBANA_STATS_TYPE = 'kibana_stats';
-export const DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S = 60;
+export const DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S = 4;
```

Comment:
I wonder if it's worth delaying 4s for every telemetry request when there's a constantly failing collector.

Some collectors have a `isReady: async () => await pluginStartObservable$.toPromise().then(() => true)`. They don't really perform any init action nor ES request in `isReady` (and they shouldn't). I'd limit this to a few ms (1s max). What do you think?